ElementX/Sources/Other/Extensions/UNNotificationContent.swift

```diff
@@ -22,6 +22,18 @@ extension UNNotificationContent {
     @objc var receiverID: String? {
         userInfo[NotificationConstants.UserInfoKey.receiverIdentifier] as? String
     }
+
+    @objc var notificationIdentifier: String? {
```

Comment:
Perhaps for consistency with the others?
```suggestion
    @objc var notificationID: String? {
```